apps/cli/src/__tests__/__examples__/action-single-quote.ts

```diff
@@ -0,0 +1,13 @@
+import { createMachine } from "xstate";
+
+createMachine({
+  tsTypes: {} as import("./action-single-quote.typegen").Typegen0,
+  schema: {
+    events: {} as { type: "PING" },
+  },
+  on: {
+    PING: {
+      actions: "say 'We surely shall see the sun shine soon'",
```

Comment:
WTF sort of British aristocrat are you, Andarist?

Reply:
one that googles "tongue twister" 🤣